code/drasil-lang/lib/Language/Drasil/Document.hs

```diff
@@ -102,13 +97,12 @@ mkParagraph x = UlC $ ulcc $ Paragraph x
 -- | Smart constructor that wraps 'LabelledContent' into 'Contents'.
 -- Takes a Reference to extract UID, LblType, and ShortName for the labelled content.
 mkFig :: Reference -> RawContent -> Contents
-mkFig r rc = LlC $ llcc (r ^. uid) (getRefAdd r) (shortname r) rc
+mkFig r rc = LlC $ LblC (r ^. uid) (Reference (r ^. uid) (getRefAdd r) (shortname r)) rc
```

Comment:
1. `mkFig` should use `llccFig`.
2. If this constructor is barely used, it should be deleted.